build-setup/windows.md

```diff
@@ -233,16 +233,40 @@ git submodule sync
 git submodule update --init --recursive
 ```
 
+In `caffe2/utils/CMakeLists.txt` delete the following line (line 4):
+
+```
+    utils/threadpool/pthreadpool-cpp.cc
+```
```

Comment:
This one is already in the PyTorch master branch, so shouldn't be necessary for us to do locally when we upgrade to version 1.8.0 - see https://github.com/pytorch/pytorch/pull/45610/files#diff-7a428c42f928ff643f0eae8a0b095bb6af88289aae4ba334ab0a62ce617c64acL4